runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksConfig.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2018, Infinitay <https://github.com/Infinitay>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package net.runelite.client.plugins.dailytaskindicators;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+	keyName = "dailytaskindicators",
+	name = "Daily Task Indicators",
+	description = "Configuration for Daily Task Indicators plugin")
+public interface DailyTasksConfig extends Config
+{
+	@ConfigItem(
+		keyName = "showHerbBoxes",
+		name = "Show Herb Boxes",
+		description = "Configures whether or not to show a message when you can" +
+			" <br/>collect your daily herb boxes at NMZ",
+		position = 1
+	)
+	default boolean showHerbBoxes()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "showStaves",
+		name = "Show Claimable Staves",
+		description = "Configures whether or not to show a message when you can" +
+			" <br/>collect your daily staves from Zaff",
```

Comment:
Based on Discord discussion, this should probably be moved to ConfigPanel (e.g it will limit the tooltips at X characters and then automatically add `<br />`